JsonFx/JsonFx.Json.UnitTests/SurrogateTest.cs

```diff
@@ -10,13 +10,31 @@ namespace JsonFx.Json
     [TestFixture()]
     class SurrogateTest
     {
-        	
 		[Test()]
-		public void TestSurrogate()
+		public void TestSurrogates()
```

Comment:
Would it be better to split this monolithic test into a case for each specific facet we're covering so that its explicitly clear what is wrong from the test results?